inc/class-statify-dashboard.php

```diff
@@ -191,7 +191,7 @@ public static function print_backview() {
 
 		// Load view.
 		load_template(
-			wp_normalize_path( sprintf( '%s/views/widget-back.php', STATIFY_DIR ) )
+			wp_normalize_path( sprintf( '%s/views/widget_back.view.php', STATIFY_DIR ) )
```

Comment:
This line should not be changed, the file was renamed to `widget-back.php`, so that's correct. Seems that merge conflicts weren't resolved correctly.

Reply:
`days_show` option was omitted in `sanitize_options()`, should be fixed by the following commit.